doc/whatsnew/2.0.rst

```diff
@@ -89,6 +89,29 @@ New checkers
           def not_useless_1(self, first, *args):
               return super(Impl, self).not_useless_1(first + some_value, *args)
 
+* A new warning was added, ``len-as-condition``, which is used whenever
+  we detect that a condition uses ``len(SEQUENCE)`` incorrectly. Instead
+  one could use ``if SEQUENCE`` or ``if not SEQUENCE``.
+
+  For instance, all of the examples below are incorrect.
+
+  .. code-block:: python
+
+      if len(S):
+        pass
+
+      if not len(S):
+        pass
+
+      if len(S) > 0:
+        pass
+
+      if len(S) != 0:
+        pass
+
+      if len(S) == 0:
+        pass
+
```

Comment:
Consider adding an example of how to correctly handle this and linking to https://www.python.org/dev/peps/pep-0008/#programming-recommendations for our less experienced python developer friends 😄